packages/xs-extra/vhd-tool.master/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "xen-api@lists.xen.org"
+authors: [ "xen-api@lists.xen.org" ]
+homepage: "https://github.com/xapi-project/vhd-tool"
+bug-reports: "https://github.com/xapi-project/vhd-tool/issues"
+dev-repo: "https://github.com/xapi-project/vhd-tool.git"
+tags: [
+  "org:mirage"
+  "org:xapi-project"
+]
+build: [
+  ["oasis" "setup"]
+  ["./configure" "--bindir=%{bin}%" "--etcdir=%{etc}%" "--libexecdir=%{bin}%"]
+  [make]
+]
+depends: [
+  "ocamlfind"
+  "lwt" {>= "2.4.3"}
+  "cstruct" {>= "1.0.1"}
+  "vhd-format" {>= "0.7.0"}
+  "uuidm"
+  "cmdliner"
+  "nbd" {>= "1.0.1"}
+  "ounit"
+  "uri"
+  "tar-format"
+  "sha"
+  "cohttp" {>="0.12.0"}
```

Comment:
An upper bound is needed here as well < "0.22.0": http://rgrinberg.com/posts/cohttp-breakage-ahead/